specification/cognitiveservices/Language.AnalyzeText/models/common.models.tsp

```diff
@@ -485,9 +485,17 @@ model PreBuiltTaskParameters is TaskParameters {
   modelVersion?: string = "latest";
 }
 
+alias AnalyzeTextBody = {
+  /** The input documents to analyze. */
+  @body
+  @clientName("analyzeTextInput", "csharp")
+  body: AnalyzeTextTask;
+};
+
 alias ShowStatsQueryParameter = QueryParametersTrait<{
   /** (Optional) if set to true, response will contain request and document level statistics. */
   @query
+  @clientName("showStatistics", "csharp")
```

Comment:
Same for this instance and any other `@clientName` usage that's outside of client.tsp

Reply:
This one I tried but updating for an alias in the client.tsp did not work if I remember correctly. I can try once more

Reply:
<img width="590" alt="image" src="https://github.com/user-attachments/assets/b97a78bb-8b84-4edd-b1c7-e2c2deb3dc92">